build-tools/packages/build-cli/test/commands/generate/buildVersion.test.ts

```diff
@@ -436,6 +440,81 @@ describe("generate:buildVersion", () => {
 			expect(ctx.stdout).to.contain("version=2.0.0-rc.7.0.0");
 			expect(ctx.stdout).to.contain("isLatest=false");
 		});
+
+	test
+		.env({
+			VERSION_BUILDNUMBER: "212045",
+			VERSION_TAGNAME: "client",
+			TEST_BUILD: "false",
+			VERSION_RELEASE: "release",
+			VERSION_INCLUDE_INTERNAL_VERSIONS: "False",
+		})
+		.stdout()
+		.command(["generate:buildVersion", "--fileVersion", "2.0.0", "--tags", ...test_tags])
+		.it("2.0 version, release", (ctx) => {
+			expect(ctx.stdout).to.contain("version=2.0.0");
```

Comment:
Not sure how `.contain()` works here, if it's looking for substrings, or exact matches in an array of output lines in stdout... would this pass even if the output incorrectly said `version=2.0.0-212045`?

Reply:
Excellent catch. I refactored it to address this but I don't love my solution. This was indeed hiding a few things, but
think the tests can just be updated and no product changes are needed.